main.py

```diff
@@ -356,6 +364,7 @@ def play_episode(self, episode_id):
         return {
             'title': episode.title,
             'podcast_title': episode.podcast.title,
+            'podcast_coverart': self._get_cover(episode.podcast),
```

Comment:
why is this added ?

Reply:
I suspect that was added as part of some other work for showing coverart a while ago, I'm puzzled that it isn't in upstream already.

Need to still check what happened there exactly but I also suspect that leaving it is not a bad thing since it is something that is being used afaik.

Reply:
After re-checking I at a total loss why that would be considered an addition, it was added in https://github.com/gpodder/gpodder-ui-qml/pull/8/files#diff-5bc02cefb3ea9e27f1a6776eabd1935d

Reply:
OK. This is because your branch is based on an older version of master, where you don't have the merged #8 code.